app/api/server/api.d.ts

```diff
@@ -3,12 +3,12 @@ import type { IUser } from '../../../definition/IUser';
 import { IMethodConnection } from '../../../definition/IMethodThisType';
 import { ITwoFactorOptions } from '../../2fa/server/code';
 
-type SuccessResult<T> = {
+export type SuccessResult<T> = {
```

Comment:
You shouldn't need to export these types, were you having any issues? 🤔

Reply:
That's right. I later found I that there's no reason whatsoever to use these types. Will fix it now...